.clang-format-ignore

```diff
@@ -1,3 +1,8 @@
+cpp/include/generated
+
```

Comment:
There are also generated files in `cpp/src/*/generated`

Reply:
Yep, I grouped them into 2 categories!